l2gersync/migrations/lastgersync0003.sql

```diff
@@ -0,0 +1,5 @@
+-- +migrate Down
+ALTER TABLE imported_global_exit_root DROP COLUMN block_pos;
+
+-- +migrate Up
+ALTER TABLE imported_global_exit_root ADD block_pos INTEGER NULL;
```

Comment:
Thinking out loud, do we need a way to update existing events in the `db`, or since you changed the file name, maybe we need to resync from scratch?

Reply:
I eas talking about the default db file name. If someone was using the old db, and it didnt specify the name, we will create a new one with new name, which will make it resync everything from scratch, which is good, so we can have block position populated.